lib/google_maps.ex

```diff
@@ -851,6 +851,112 @@ defmodule GoogleMaps do
     GoogleMaps.get("place/queryautocomplete", params)
   end
 
+@doc """
+  Search for nearby places based on location and keywords
+
+  `nearby_search/3/4` - Most common use for Places API.
+
+  `nearby_search/1/2` - Intended for use with `rankby` 
+    when you cant use radius.
+
+## Args:
+
+    * `location` — The latitude/longitude around which to
+      retrieve place information. This must be specified as
+      *latitude,longitude*.
+
+    * `radius` — Defines the distance (in meters) within which
+      to return place results. The maximum allowed radius is 50 000 meters.
+      Note that radius must not be included if rankby=distance
+      (described under Optional parameters below) is specified
+
+    * `keyword` — The text string on which to search. The Places
+      service will return candidate matches based on this
+      string and order results based on their perceived relevance.
+
+  ## Options:
+
+    * `language` — The language code, indicating in which language the
+      results should be returned, if possible. Searches are also biased
+      to the selected language; results in the selected language may be
+      given a higher ranking. See the [list of supported languages](https://developers.google.com/maps/faq#languagesupport)
+      and their codes. Note that we often update supported languages so
+      this list may not be exhaustive. If language is not supplied, the
+      Places service will attempt to use the native language of the
+      domain from which the request is sent.
+
+    * `minprice` and `maxprice` - Restricts results to only those places 
+      within the specified price level. Valid values are in the range 
+      from 0 (most affordable) to 4 (most expensive), inclusive. 
+      The exact amount indicated by a specific value will vary from 
+      region to region.
+
+    * `opennow` - Returns only those places that are open for business at
+      the time the query is sent. Places that do not specify opening hours
+      in the Google Places database will not be returned if you include
+      this parameter in your query.
+
+    * `region` - The region code, specified as a ccTLD (country code top-level domain)
```

Comment:
I don't see `region` on https://developers.google.com/places/web-service/search#PlaceSearchRequests

Reply:
`Region` was under the `textsearch` docs, I just scrolled down too far in their docs 😄 . Good catch!